static/app/types/debugFiles.tsx

```diff
@@ -41,5 +41,15 @@ export type AppStoreConnectValidationData = {
   itunesSessionRefreshAt: string | null;
   /** Indicates if the itunesSession is actually *needed* to complete any downloads that are pending. */
   pendingDownloads: number;
+  /**
+   * The version of the latest build recognized by sentry. The contents of this string is just a
+   * number. This will be null if no builds can be found.
```

Comment:
```suggestion
   * The build number of the latest build recognized by sentry.  This does not imply the dSYMs for this
   * build have been fetched. The contents of this string is just a number. This will be null if no builds
   * can be found.
```